src/sql-parser/src/ast/defs/ddl.rs

```diff
@@ -202,6 +201,33 @@ impl<T: AstInfo> Format<T> {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+pub enum CsvColumns {
```

Comment:
NIT: it'd be nice if we picked one enum structure for both `CsvColumns` and `ColumnSpec` like using `HeaderNames` here or `Header { names` there